R/redist_map.R

```diff
@@ -9,8 +9,10 @@
 # constructors and reconstructors
 
 # Main internal constructor
-new_redist_map <- function(data, adj, ndists, pop_bounds, pop_col = "pop",
-                           adj_col = "adj", add_adj = TRUE, existing_col = NULL) {
+new_redist_map <- function(
+        data, adj, ndists, total_seats, district_seat_sizes,
```

Comment:
@philipwosull 
rename to `nseats` for consistency and `seats_range`
let's default `nseats=ndists` and `seats_range=c(1, 1)` or `1` or whatever the format is

Reply:
Ah, sorry, this is the internal constructor. The comment about defaults applies to `redist_map` below